docs/compilers/CSharp/Runtime Async Design.md

```diff
@@ -301,23 +311,44 @@ class C
 }
 ```
 
+Translated C#:
```

Comment:
Could we reduce the number of sections by removing the now unnecessary distinction between awaiting a call and awaiting a local? We could have a single case for awaiting an expression of type Task/Task<T>/ValueTask/ValueTask<T>.